pkgs/applications/science/electronics/verilog/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "iverilog-${version}";
-  version = "2017.08.12";
+  version = "2018.12.15";
```

```suggestion
  version = "unstable-2018.12.15";
```
https://nixos.org/nixpkgs/manual/#sec-package-naming

---

Not that there appear to be any consistency within Nixpkgs but shouldn't `unstable` be part of (p)name to more closely reflect the manual guidelines?

---

(And it kind of renames an existing package, if not its attribute name?..)

---

Yes, this would also break any lexicographic sorting/version comparisons on `.version` as well (`builtins.compareVersions`, etc). Such a rename also will break people who have installed by the existing attribute name.

But I don't think `-unstable` needs to be part of the name in any case (most of the other EDA-oriented expressions, many of which I maintain, use this same convention without `-unstable`). This is fine to merge.